### PR TITLE
updated docs

### PR DESCRIPTION
**Pull request by katunin-ng**
Before submitting a pull request, Please make sure the following:

## Description

Please explain the changes you made.

Link to design (if applicable)

### Checklist
- [ ] Code compiles correctly
- [ ] Relevant tests were added (unit / contract / integration)
- [ ] Relevant logs were added
- [ ] Formating and linting run locally successfully
- [ ] All tests passing
- [x ] UA review
- [ ] Design is documented
- [x] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ development descriptor (`mta.yaml` file) or from module build artifacts accordin
 ## Contributions
 
 Contributions are greatly appreciated.
-If you want to contribute, please follow [the guidelines](./.github/CONTRIBUTING.md).
+If you want to contribute, please follow [the guidelines](docs/docs/process.md).
```
Comment (jhornik) [inline README.md]: If you want to contribute, follow [the guidelines](docs/docs/process.md).

File: docs/docs/migration.md [hunk 2]
```diff
@@ -14,21 +14,21 @@ You need to update your `mta.yaml` file to exclude `html5` modules from the resu
...
-<br>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; For more information about the `supported-platforms` build parameter, see [Configuring and Packaging Modules According to Target Platforms](configuration.md#configuring-and-packaging-modules-according-to-target-platforms).
+&nbsp;
 * The following `build-parameters` are not supported by the Cloud MTA Build Tool: <ul><li>`npm-opts`<li>`grunt-opt`<li>`maven-opts`</ul>
 
-  If you need to change the default build behavior defined for the correponding builder, see [configure `custom` builder](configuration.md#configuring-the-custom-builder).
-  For a complete list of available builders and their default behaviors, see [Builders execution commands](https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/builder_type_cfg.yaml).
-  <br>
+  If you need to change the default build behavior defined for the correponding builder, see [configure `custom` builder](configuration.md#configuring-the-custom-builder). For a complete list of available builders and their default behaviors, see [Builders execution commands](https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/builder_type_cfg.yaml).
```
Comment (jhornik) [inline docs/docs/migration.md]: If you need to change the default build behavior defined for the corresponding builder, see [configure `custom` builder](configuration.md#configuring-the-custom-builder). For a complete list of available builders and their default behaviors, see [Builders execution commands](https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/builder_type_cfg.yaml).

[typo in "corresponding"]

File: docs/docs/migration.md [hunk 3]
```diff
@@ -14,21 +14,21 @@ You need to update your `mta.yaml` file to exclude `html5` modules from the resu
...
+  &nbsp;
 
 #### New features in the Cloud MTA Build Tool
 
-* If you want to run a builder process before running builders of the specific modules, define it using [global `before-all` build parameters](configuration.md#configuring-global-build).
-
-<br>
-* You can define your own build commands as described here: [configuring `custom` builder](configuration.md#configuring-the-custom-builder).
-<br>
+* In addition to configuring build behaviour in `mta.yaml`, you can configure build process of the specific module or the whole project in the `Makefile.mta` file that you can generate the file using [`mbt init` command](usage.md#cloud-mta-build-tool-commands). The generated file contains default configurations for buidling MTA project according to our best practices.
```
Comment (jhornik) [inline docs/docs/migration.md]: In addition to configuring the build behavior in the `mta.yaml` file, you can configure the build process of the specific module or the whole project in the `Makefile.mta` file, which you can generate using the [`mbt init` command](usage.md#cloud-mta-build-tool-commands). The generated file contains the default configurations for building the MTA project according to our best practices.

File: README.md (hunk 1 above)
Comment (katunin-ng) [reply README.md]: Updated

File: docs/docs/migration.md (hunk 2 above)
Comment (katunin-ng) [reply docs/docs/migration.md]: Fixed

File: docs/docs/migration.md (hunk 3 above)
Comment (katunin-ng) [reply docs/docs/migration.md]: Done

File: README.md (hunk 1 above)
Comment (jhornik) [reply README.md]: Thanks

File: docs/docs/migration.md (hunk 2 above)
Comment (jhornik) [reply docs/docs/migration.md]: Thanks

File: docs/docs/migration.md (hunk 3 above)
Comment (jhornik) [reply docs/docs/migration.md]: Thanks

**Pull request merged**